src/test/java/com/ixortalk/organization/api/rest/UserRestController_ResendInvite_IntegrationAndRestDocTest.java

```diff
@@ -105,7 +95,29 @@ public void organizationAdminResendInvite() throws JsonProcessingException {
                                                         convertToHowItShouldBeSentToMailingService(userInOrganizationXInvited),
                                                         IMAGE_DOWNLOAD_URL_PREFIX + organizationX.getLogo(), true,
                                                         userRestResource.findById(userInOrganizationXInvited.getId()).get().getAcceptKey().getAcceptKey()))))));
+    }
+
+    @Test
+    public void organizationAdminResendInviteForUserWithInviteLanguageNl() throws JsonProcessingException {
```

Comment:
Test name is not correct according to what is tested (`nl` vs. `en`).  Also remove exception that is not being thrown.